src/main/plugins/org.dita.pdf2/cfg/common/vars/zh_TW.xml

```diff
@@ -0,0 +1,344 @@
+
+    <!-- Text to use for appendix titles.-->
+    <variable id="Appendix with number">附錄 <param ref-name="number"/></variable>
+
+    <!-- Text to use for part titles.-->
+    <variable id="Part with number">第 <param ref-name="number"/> 篇 </variable>
+
+    <!-- Text to use for a label generated from <required-cleanup> element. -->
+    <variable id="Required-Cleanup">[必要修正]</variable>
+
+    <!-- Template for generated page number of a reference 
+         in printed manual. -->
+    <variable id="On the page"> 頁 <param ref-name="pagenum"/></variable>
+
+    <!-- Template for generated page number of a reference
+         in printed manual, when referenced element has no title. -->
+    <variable id="Page"> 頁 <param ref-name="pagenum"/></variable>
+
+    <!-- UNUSED: Template for generated page number of a reference
+         in chm/html, when referenced element has no title. -->
+    <variable id="This link"></variable>
+
+    <!-- Image path to use for a note of "note" type. -->
+    <variable id="note Note Image Path"></variable>
+    <!-- Image path to use for a note of "notice" type. -->
+    <variable id="notice Note Image Path"></variable>
+    <!-- Image path to use for a note of "attention" type. -->
+    <variable id="attention Note Image Path">Configuration/OpenTopic/cfg/common/artwork/warning.gif</variable>
+
+    <!-- Image path to use for a note of "caution" type. -->
+    <variable id="caution Note Image Path">Configuration/OpenTopic/cfg/common/artwork/warning.gif</variable>
+
+    <!-- Image path to use for a note of "danger" type. -->
+    <variable id="danger Note Image Path">Configuration/OpenTopic/cfg/common/artwork/warning.gif</variable>
+    <!-- Image path to use for a note of "warning" type. -->
+    <variable id="warning Note Image Path">Configuration/OpenTopic/cfg/common/artwork/warning.gif</variable>
+    <!-- Image path to use for a note of "fastpath" type. -->
+    <variable id="fastpath Note Image Path"></variable>
+
+    <!-- Image path to use for a note of "important" type. -->
+    <variable id="important Note Image Path"></variable>
+
+    <!-- Image path to use for a note of "remember" type. -->
+    <variable id="remember Note Image Path"></variable>
+
+    <!-- Image path to use for a note of "restriction" type. -->
+    <variable id="restriction Note Image Path"></variable>
+
+    <!-- Image path to use for a note of "tip" type. -->
+    <variable id="tip Note Image Path"></variable>
+    <variable id="trouble Note Image Path">Configuration/OpenTopic/cfg/common/artwork/warning.gif</variable>
+    <!-- Image path to use for a note of "other" type. -->
+    <variable id="other Note Image Path"></variable>
+
+    <!-- UNUSED: The title of the links to child topics -->
+    <variable id="Child topics"></variable>
+
+    <!-- UNUSED: The title of the links to Related references -->
+    <variable id="Related references"></variable>
+
+    <!--
+       UNUSED IN PDF2: Strings for online-help publishing.
```

Comment:
I was thinking I would do a single commit / pull request that marked these "deprecated" across all languages, then remove from every language in 3.0. I don't think there's a clear best practice about adding them in these 4 languages. I added them in all of the other languages for 2.2 / 2.3 just because they were part of English, and it seemed if they're in English they should be present in others.